### DISCUSSION TITLE
With SIGNUPS_ALLOWED=false how do I create my own account?

### DISCUSSION THREAD

**Discussion by saugatadatta** [dani-garcia/vaultwarden#2387] [Q&A]
I have deployed with SIGNUPS_ALLOWED=false, now How do I create my own account?, I have added local self sign certificate via Nginx reverse proxy, and HTTPS issue resolved, now I ma bit confused.

**Comment by RealOrangeOne** [comment]
I think you can create users through the admin (https://github.com/dani-garcia/vaultwarden/wiki/Enabling-admin-page). If not, just temporarily allow signups, create your user, then disable it again.

**Reply by BlackDex** [reply]
You can via the /admin interface no need to allow and disallow this.

**Comment by saugatadatta** [comment]
After creation of the container with SIGNUPS_ALLOWED=true and created the account , re-deployment with SIGNUPS_ALLOWED=false will work.
